API/API.ml

```diff
@@ -180,7 +180,27 @@ module Prelude =
    there is no "kernel/entries.ml" file *)
 module Entries =
   struct
-    type mutual_inductive_entry = Entries.mutual_inductive_entry
+  type local_entry = Entries.local_entry =
```

Comment:
I'll try to [fix formatting](https://github.com/matejkosik/coq/blob/6c384c49404ee5ed3af1679a76ad2aeb4d383dc2/API/PROBLEMS/uncategorized#L51) when there will be a good opportunity (no PRs that would be broken).